src/tests/suite/daos_obj.c

```diff
@@ -3437,6 +3592,10 @@ static const struct CMUnitTest io_tests[] = {
 	  split_sgl_update_fetch, async_disable, test_case_teardown},
 	{ "IO36: trigger server pool map refresh",
 	  io_pool_map_refresh_trigger, async_disable, test_case_teardown},
+	{ "IO37: Fetch nonexistent keys",
+	  fetch_nonexistent_keys, async_disable, test_case_teardown},
+	{ "IO38: Fetch mixed existing/nonexistent keys",
```

Comment:
"Fetch mixed existing and nonexistent akeys in single fetch call".

Reply:
I will update the code and test name